texi2rst.py

```diff
@@ -1658,6 +1738,20 @@ def test_subsubheading(self):
         self.assertEqual(u'A sub-sub-heading\n^^^^^^^^^^^^^^^^^\n\nsome text\n\n',
                          out)
 
+    def test_option_ref_with_var(self):
+        xml_src = '<para>This is similar to how <option>-Walloca-larger-than=</option><var>byte-size</var> works.</para>'
+        doc = from_xml_string(xml_src)
+        doc = convert_to_rst(doc, self.ctxt)
+        out = self.make_rst_string(doc)
+        self.assertEqual('This is similar to how :option:`-Walloca-larger-than` ``=byte-size`` works.\n\n', out)
```

Comment:
Is there a way to lose the space between the option and the `=` and have the .rst still be parsable?

Reply:
I don't think so, apparently, any touching \`something\`\`something_else\` leads to a problem. That's why I added the fixup pass that adds such spaces..

Reply:
Fixed now `:samp:`.